cadence/contracts/GametvNFT.cdc

```diff
@@ -183,8 +181,17 @@ pub contract GametvNFT: NonFungibleToken {
     pub resource NftTypeHelper : NftTypeHelperPublic {
         // public function to borrow details of NFTtype
         pub fun borrowNFTtype(id: String): NftType? {
```

Comment:
Is there a reason you did not name this borrowActiveNFTtype, using the convention you followed below for borrowStaleNFTtype?